textile/chat-features.textile

```diff
@@ -591,6 +591,10 @@ This section contains error codes that are specific to Chat. If a specific error
 For non-chat-specific codes, the status code for the error should align with the error code. For example, error code @40000@ should hav status code @400@.
 
 <pre>
+  // The message was rejected before publishing by a rule on the chat room.
+  // For example, a message may be rejected because it violates a moderation policy that bans profanity.
+  MessageRejectedByBeforePublishRule = 42210
```

Comment:
Is this supposed to be [42211](https://github.com/ably/realtime/blob/6c4cc67833bf4228834da8f493031684b987cd04/go/realtime/lib/errors/constants.go#L25)?

Reply:
Yes - fixed and added 42213.